### PR TITLE
add: `firefox-arm64-bin`

### PR DESCRIPTION
**Pull request by oklopfer**

### CODE REVIEW

**Comment by oklopfer** [comment]
apparently the tar hash isn't the package hash, gotta go find it rq

File: packages/firefox-arm64-bin/firefox-arm64-bin.pacscript [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+repology=("project: ${gives}")
+build_depends="libarchive-tools"
+url="http://mirror.archlinuxarm.org/aarch64/extra/${gives}-${version}-aarch64.pkg.tar.xz"
+description="Free and open-source web browser developed by the Mozilla Foundation"
+maintainer="Oren Klopfer <oren@taumoda.com>"
+hash="e457841208d1da863bc0d2ae0b0045b117c5e5a69479c9326176e04ce4e0a185"
+
+build() {
+    true
+}
```
Comment (Henryws) [inline packages/firefox-arm64-bin/firefox-arm64-bin.pacscript:13]:
```suggestion
```

File: packages/firefox-arm64-bin/firefox-arm64-bin.pacscript [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+install() {
+    sudo install -Dm755 "usr/bin/firefox" -t "${pkgdir}/usr/bin/firefox"
+    sudo cp -r "usr/lib/firefox" "${pkgdir}/usr/lib/"
+    sudo install -Dm755 "usr/share/applications/firefox.desktop" -t "${pkgdir}/usr/share/applications/firefox-stable.desktop"
+    sudo install -Dm644 "usr/share/icons/hicolor/16x16/apps/firefox.png" -t "${pkgdir}/usr/share/icons/hicolor/16x16/apps/firefox.png"
```
Comment (Henryws) [inline packages/firefox-arm64-bin/firefox-arm64-bin.pacscript:19]:
```suggestion
    sudo install -Dm644 "usr/share/icons/hicolor/" -t "${pkgdir}/usr/share/icons/"
```

File: packages/firefox-arm64-bin/firefox-arm64-bin.pacscript [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    sudo install -Dm644 "usr/share/icons/hicolor/24x24/apps/firefox.png" -t "${pkgdir}/usr/share/icons/hicolor/24x24/apps/firefox.png"
+    sudo install -Dm644 "usr/share/icons/hicolor/32x32/apps/firefox.png" -t "${pkgdir}/usr/share/icons/hicolor/32x32/apps/firefox.png"
+    sudo install -Dm644 "usr/share/icons/hicolor/48x48/apps/firefox.png" -t "${pkgdir}/usr/share/icons/hicolor/48x48/apps/firefox.png"
+    sudo install -Dm644 "usr/share/icons/hicolor/64x64/apps/firefox.png" -t "${pkgdir}/usr/share/icons/hicolor/64x64/apps/firefox.png"
+    sudo install -Dm644 "usr/share/icons/hicolor/128x128/apps/firefox.png" -t "${pkgdir}/usr/share/icons/hicolor/128x128/apps/firefox.png"
+    sudo install -Dm644 "usr/share/icons/hicolor/192x192/apps/firefox.png" -t "${pkgdir}/usr/share/icons/hicolor/192x192/apps/firefox.png"
+    sudo install -Dm644 "usr/share/icons/hicolor/256x256/apps/firefox.png" -t "${pkgdir}/usr/share/icons/hicolor/256x256/apps/firefox.png"
+    sudo install -Dm644 "usr/share/icons/hicolor/384x384/apps/firefox.png" -t "${pkgdir}/usr/share/icons/hicolor/384x384/apps/firefox.png"
```
Comment (Henryws) [inline packages/firefox-arm64-bin/firefox-arm64-bin.pacscript:28]:
```suggestion
```

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/firefox-arm64-bin/firefox-arm64-bin.pacscript [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+description="Free and open-source web browser developed by the Mozilla Foundation"
+maintainer="Oren Klopfer <oren@taumoda.com>"
+hash="e457841208d1da863bc0d2ae0b0045b117c5e5a69479c9326176e04ce4e0a185"
+
+install() {
+    cd ..
+    sudo install -Dm755 "usr/bin/firefox" -t "${pkgdir}/usr/bin/firefox"
+    sudo cp -r "usr/lib/firefox" "${pkgdir}/usr/lib/"
+    sudo install -Dm755 "usr/share/applications/firefox.desktop" -t "${pkgdir}/usr/share/applications/firefox-stable.desktop"
+    sudo install -Dm644 "usr/share/icons/hicolor/16x16/apps/firefox.png" -t "${pkgdir}/usr/share/icons/hicolor/16x16/apps/firefox.png"
```
Comment (Henryws) [inline packages/firefox-arm64-bin/firefox-arm64-bin.pacscript:15]: Fix these. Find some way to compress this into one line.

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/firefox-arm64-bin/firefox-arm64-bin.pacscript [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+install() {
+    cd ..
+    sudo install -Dm755 "usr/bin/firefox" -t "${pkgdir}/usr/bin/firefox"
+    sudo cp -r "usr/lib/firefox" "${pkgdir}/usr/lib/"
+    sudo install -Dm755 "usr/share/applications/firefox.desktop" -t "${pkgdir}/usr/share/applications/firefox-stable.desktop"
+    sudo install -Dm644 "usr/share/icons/hicolor/scalable/apps/firefox.svg" -t "${pkgdir}/usr/share/icons/hicolor/scalable/apps/firefox.svg"
+    sudo install -Dm644 "usr/share/icons/hicolor/symbolic/apps/firefox-symbolic.svg" -t "${pkgdir}/usr/share/icons/hicolor/symbolic/apps/firefox-symbolic.svg"
+    for i in 16x16 22x22 24x24 32x32 48x48 64x64 128x128 192x192 256x256 384x384
+    do
```
Comment (Henryws) [inline packages/firefox-arm64-bin/firefox-arm64-bin.pacscript:18]: Put this on the line above, to conform to our style.

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/firefox-arm64-bin/firefox-arm64-bin.pacscript [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+install() {
+    cd ..
+    sudo install -Dm755 "usr/bin/firefox" -t "${pkgdir}/usr/bin/firefox"
+    sudo cp -r "usr/lib/firefox" "${pkgdir}/usr/lib/"
+    sudo install -Dm755 "usr/share/applications/firefox.desktop" -t "${pkgdir}/usr/share/applications/firefox-stable.desktop"
+    sudo install -Dm644 "usr/share/icons/hicolor/scalable/apps/firefox.svg" -t "${pkgdir}/usr/share/icons/hicolor/scalable/apps/firefox.svg"
+    sudo install -Dm644 "usr/share/icons/hicolor/symbolic/apps/firefox-symbolic.svg" -t "${pkgdir}/usr/share/icons/hicolor/symbolic/apps/firefox-symbolic.svg"
+    for i in 16x16 22x22 24x24 32x32 48x48 64x64 128x128 192x192 256x256 384x384; do
+    sudo install -Dm644 "usr/share/icons/hicolor/$i/apps/firefox.png" -t "${pkgdir}/usr/share/icons/hicolor/$i/apps/firefox.png"; done
```
Comment (Henryws) [inline packages/firefox-arm64-bin/firefox-arm64-bin.pacscript:18]:
```suggestion
    sudo install -Dm644 "usr/share/icons/hicolor/$i/apps/firefox.png" -t "${pkgdir}/usr/share/icons/hicolor/$i/apps/firefox.png"
done
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Review by Henryws — APPROVED** [review]

**Pull request merged**